.gitignore

```diff
@@ -1,2 +1,7 @@
 .DS_Store
 .idea/
+
+# magento container files
+docker-container/magento-cloud
+auth.json
```

Comment:
when adding files which are needed for local development, please add a sample liek `auth-sample.json` so that someone knows what sort of data to stick in there easily.

look at `local.env-sample` in hail or `settings-local-sample.js` in storm